CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 ### New features
 
 * [#8322](https://github.com/rubocop-hq/rubocop/pull/8322): Support autocorrect for `Style/CaseEquality` cop. ([@fatkodima][])
+* [#8376](https://github.com/rubocop-hq/rubocop/pull/8376): Add new `Lint/MissingSuper` cop. ([@fatkodima][])
```

Comment:
There should probably be a note in here that this replaces the `Style/MethodMissingSuper` cop.